### PR TITLE
Split guild and global

### LINKED ISSUE #4: Split Guild & Global commands based on ENV
When the bot is run in a `PROD` environment global commands should be initialised. Whereas when the bot is run in a `DEV` environment guild commands against a testing guild should be initialised.

[See discordjs docs](https://discordjs.guide/interactions/registering-slash-commands.html#guild-commands)

### PR DESCRIPTION
**Pull request by Puffycheeses**
Check the environment and set up either global or guild commands.

Global commands take 1 hour to propagate whereas guild commands are instant.

This resolves #4

### CODE REVIEW

**Comment by Puffycheeses** [comment]
Merge #10 beforehand 👍

File: src/index.ts [hunk 1]
```diff
@@ -38,8 +38,10 @@ client.on("interactionCreate", async (interaction) => {
 		logger.info(`Running on ${process.env.ENV} environment`);
 		await client.login(process.env.DISCORD_TOKEN);
 		logger.info(`Logged in successfully to ${client.guilds.cache.size} guilds`);
+		logger.info(
+			`https://discord.com/api/oauth2/authorize?client_id=${client.application.id}&permissions=8&scope=applications.commands%20bot`
```
Comment (zkldi) [inline src/index.ts:42]: What does this logger statement mean?

File: scripts/createEnv.sh [hunk 2]
```diff
@@ -9,6 +9,7 @@ if [[ "$ENV" != "PROD" ]]; then
   # Non-prod environment
   echo "ENV=DEV" >> "$DOT_ENV_PATH"
   echo "DISCORD_TOKEN=${NON_PROD_DISCORD_TOKEN:-"_"}" >> "$DOT_ENV_PATH"
+  echo "DEV_SERVER_ID=883949913887612938" >> "$DOT_ENV_PATH" # Where is testing taking place?
```
Comment (zkldi) [inline scripts/createEnv.sh:12]: I have a temporary discord server for testing mmind1, do you want to put it there?

File: src/index.ts (hunk 1 above)
Comment (Puffycheeses) [reply src/index.ts:42]: It's an invite link for the bot, Not 100% needed but convenient when testing. Remove?

File: scripts/createEnv.sh (hunk 2 above)
Comment (Puffycheeses) [reply scripts/createEnv.sh:12]: Yeah shoot me an invite on discord as well 👍

File: src/index.ts (hunk 1 above)
Comment (zkldi) [reply src/index.ts:42]: Maybe prefix the message with `Bot Invite Link:` ? At the moment I was just confused what it actually was :P
seems useful though, so probably worth keeping.

**Pull request merged**